<launch>
  <!-- System Monitoring -->
  <node pkg="robot_diagnostics" type="system_monitoring" respawn="false" name="system_monitoring" output="screen">
  </node>
</launch>
